server/rpc.go

```diff
@@ -73,46 +72,24 @@ func genTranscoderReq(b Broadcaster, jid int64) (*net.TranscoderRequest, error)
 	return &net.TranscoderRequest{JobId: jid, Sig: sig}, nil
 }
 
-func jobClaimable(orch Orchestrator, job *lpTypes.Job) bool {
-	if len(job.Profiles) <= 0 {
-		// This is just to be extra cautious:
-		// We don't need to do any work, so nothing to claim
-		return false
-	}
-
-	blk := orch.CurrentBlock()
-	if blk == nil {
-		// The benefit of doubt.
-		// May be offchain or have internal errors in fetching a block.
-		return true
-	}
-
-	canClaim := job.FirstClaimSubmitted || blk.Cmp(new(big.Int).Add(job.CreationBlock, big.NewInt(256))) <= 0
-
-	return canClaim && !(blk.Cmp(job.CreationBlock) == -1 || blk.Cmp(job.EndBlock) == 1)
-}
-
 func verifyMsgSig(addr ethcommon.Address, msg string, sig []byte) bool {
 	return eth.VerifySig(addr, crypto.Keccak256([]byte(msg)), sig)
 }
 
 func verifyTranscoderReq(orch Orchestrator, req *net.TranscoderRequest, job *lpTypes.Job) error {
-	if orch.Address() != job.TranscoderAddress {
+	if orch.Address() != job.TranscoderAddress { // ANGIE - NEED TO GET TRANSCODER AND BROADCASTER ADDRESS FROM SOMEWHERE ELSE
```

Comment:
Most likely PM parameters. @yondonfu  what do you think?

Reply:
Are you asking if we could get the transcoder/broadcaster address from a PM ticket? If so, at the moment, a ticket will contain the sender and recipient addresses so given a ticket this check could be:

```
if orch.Address() != ticket.RecipientAddress
```

Reply:
Thanks @yondonfu !